### PR TITLE
Use the average from the last month for the average displayed and used in calculations

### PR DESCRIPTION
**Pull request by SConaway**

### CODE REVIEW

File: api/app.js [hunk 1]
```diff
@@ -141,18 +146,16 @@ async function report() {
...
-	let thousandsTime = predictTime(thousandsGoal, latest);
+	let thousandsTime = predictTime(thousandsGoal, latest, averageSpeed);
 	let tenThousandsGoal = Math.ceil(latest / 5000) * 5000;
 	let pastThousandsGoal = Math.floor(latest / 1000) * 1000;
-	let tenThousandsTime = predictTime(tenThousandsGoal, latest);
+	let tenThousandsTime = predictTime(tenThousandsGoal, latest, averageSpeed);
 	let message =
 		`Good evening, my underlings! You have done a _brilliant_ job with the counting. Mmm. Yes. Numbers are savory. 
 		Today we've went from *${oldest}* to *${latest}*!
-		- :arrow_upper_right: The day's progress: *+${diff}*
```
Comment (avi9664) [inline api/app.js:154]: Um... I don't think we should delete that? It shows our progress that day

Comment (SConaway) [reply api/app.js:154]: yeah good idea. I was having wayyy to many issues with git for some reason. It kept deleting lines. I'll try re-adding that line through GitHub web.

Comment (SConaway) [reply api/app.js:154]: Good catch!

**Comment by SConaway** [comment]
Force Push was to fix a spelling error in the commit, sorry!

**Pull request merged**